metagpt/utils/token_counter.py

```diff
@@ -37,6 +37,8 @@
     "gpt-4-turbo": {"prompt": 0.01, "completion": 0.03},
     "gpt-4-vision-preview": {"prompt": 0.01, "completion": 0.03},  # TODO add extra image price calculator
     "gpt-4-1106-vision-preview": {"prompt": 0.01, "completion": 0.03},
+    "gpt-4o": {"prompt": 0.005, "completion": 0.015},
+    "gpt-4o-2024-05-13": {"prompt": 0.005, "completion": 0.015},
```

Comment:
This modification misses the TOKEN_MAX dictionary, please add the appropriate kv to TOKEN_MAX